pandas/tests/groupby/test_transform.py

```diff
@@ -1074,3 +1072,54 @@ def test_transform_lambda_with_datetimetz():
         name="time",
     )
     assert_series_equal(result, expected)
+
+
+def test_transform_cumcount_ngroup():
+    df = DataFrame(dict(a=[0, 0, 0, 1, 1, 1], b=range(6)))
+    g = df.groupby(np.repeat([0, 1], 3))
+
+    # GH 27472
+    result = g.transform("cumcount")
+    expected = g.cumcount()
+    assert_series_equal(result, expected)
+
+    # GH 27468
+    result = g.transform("ngroup")
+    expected = g.ngroup()
+    assert_series_equal(result, expected)
+
+
+@pytest.mark.parametrize(
+    "func",
+    [
+        "backfill",
+        "bfill",
+        "cumcount",
+        "cummax",
+        "cummin",
+        "cumprod",
+        "cumsum",
+        "diff",
+        "ffill",
+        "pad",
+        "pct_change",
+        "rank",
+        "shift",
+        "ngroup",
+        pytest.param("fillna", marks=pytest.mark.xfail(reason="TODO: potential bug")),
```

Comment:
Can you find / open GitHub issues for each of these and refer to them in the reason?

Reply:
yeah, actually I am not sure this is a bug, so add `potential` in front. Actually, there are some functions in `transform` that never got tested, those three are all among untested funcs. I am working on a pr now to add tests for those cases, since this PR is yet finished, I was supposed to wait for the review in this PR and afterwards submit the other PR. But I will of course open issue and refer those three to the new issue!